# References

1. Gasiunas G, Young JK, Karvelis T, Kazlauskas D, Urbaitis T, Jasnauskaite M, et al. A catalogue of biochemically diverse CRISPR-Cas9 orthologs. *Nat Commun.* 2020.
2. Wei J, Hou L, Liu J, Wang Z, Gao S, Qi T, et al. Closely related type II-C Cas9 orthologs recognize diverse PAMs. *Elife.* 2022.
3. Karvelis T, Gasiunas G, Young J, Bigelyte G, Silanskas A, Cigan M, et al. Rapid characterization of CRISPR-Cas9 protospacer adjacent motif sequence elements. *Genome Biol.* 2015.
4. Liu Z, Chen S, Xie W, Yu H, Lai L, Li Z. Versatile and efficient genome editing with Neisseria cinerea Cas9. *Communications Biology.* 2022.
5. Hand T, Das A, Li H. Directed evolution studies of a thermophilic Type II-C Cas9. *Methods in Enzymology.* Academic Press; 2019.
6. Hirano H, Gootenberg JS, Horii T, Zhang F, Nishimasu H, Nureki O. Structure and Engineering of Francisella novicida Cas9. *Cell.* 2016.
7. Cui Z, Tian R, Huang Z, Jin Z, Li L, Liu J, et al. FrCas9 is a CRISPR/Cas9 system with high editing efficiency and fidelity. *Nat Commun.* 2022.
8. Kim E, Koo T, Park SW, Kim D, Kim K, Cho HY, et al. In vivo genome editing with a small Cas9 orthologue derived from *Campylobacter jejuni.* *Nat Commun.* 2017.
9. Hirano S, Abudayyeh OO, Gootenberg JS, Horii T, Ishitani R, Hatada I, et al. Structural basis for the promiscuous PAM recognition by *Corynebacterium diphtheriae* Cas9. *Nat Commun.* 2019.
10. Zetsche B, Gootenberg JS, Abudayyeh OO, Slaymaker IM, Makarova KS, Essletzbichler P, et al. Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR-Cas system. *Cell.* 2015.
11. Zetsche B, Abudayyeh OO, Gootenberg JS, Scott DA, Zhang F. A Survey of Genome Editing Activity for 16 Cas12a Orthologs. *Keio J Med.* 2020.
12. Strecker J, Jones S, Koopal B, Schmid-Burgk J, Zetsche B, Gao L, et al. Engineering of CRISPR-Cas12b for human genome editing. *Nat Commun.* 2019.
13. Harrington LB, Ma E, Chen JS, Witte IP, Gertz D, Paez-Espino D, et al. A scoutRNA Is Required for Some Type V CRISPR-Cas Systems. *Mol Cell.* 2020.
14. Karvelis T, Bigelyte G, Young JK, Hou Z, Zedaveinyte R, Budre K, et al. PAM recognition by miniature CRISPR–Cas12f nucleases triggers programmable double-stranded DNA target cleavage. *Nucleic Acids Res.* 2020.
15. Yan WX, Hunnewell P, Alfonse LE, Carte JM, Keston-Smith E, Sothiselvam S, et al. Functionally diverse type V CRISPR-Cas systems. *Science.* 2019.
16. Wang Y, Qi T, Liu J, Yang Y, Wang Z, Wang Y, et al. A highly specific CRISPR-Cas12j nuclease enables allele-specific genome editing. *Sci Adv.* 2023.
17. Strecker J, Ladha A, Gardner Z, Schmid-Burgk JL, Makarova KS, Koonin EV, et al. RNA-guided DNA insertion with CRISPR-associated transposases. *Science.* 2019.
18. Urbaitis T, Gasiunas G, Young JK, Hou Z, Paulraj S, Godliauskaite E, et al. A new family of CRISPR-type V nucleases with C-rich PAM recognition. *EMBO Rep.* 2022.
19. Wu WY, Mohanraju P, Liao C, Adiego-Pérez B, Creutzburg SCA, Makarova KS, et al. The miniature CRISPR-Cas12m effector binds DNA to block transcription. *Mol Cell.* 2022.
20. Al-Shayeb B, Skopintsev P, Soczek KM, Stahl EC, Li Z, Groover E, et al. Diverse virus-encoded CRISPR-Cas systems include streamlined genome editors. *Cell.* 2022.
21.	Burstein D, Harrington L, Strutt S, Probst A, Anantharaman K, Thomas B, Doudna J, and Banfield J. New crispr–cas systems from uncultivated microbes. *Nature* 2017. 